integrationtest/src/de/tobiasroeser/mill/integrationtest/MillIntegrationTestModule.scala

```diff
@@ -67,29 +68,20 @@ trait MillIntegrationTestModule extends TaskModule {
     // publish Local
     val ivyPath = ctx.dest / "ivyRepo"
 
-    ctx.log.debug("Publishing plugins under test into test ivy repo")
-    val publisher = new LocalIvyPublisher(ivyPath / 'local)
-    (pluginUnderTestDetails() ++ temporaryIvyModulesDetails()).foreach { detail =>
-      publisher.publish(
-        jar = detail._1.path,
-        sourcesJar = detail._2._1.path,
-        docJar = detail._2._2._1.path,
-        pom = detail._2._2._2._1.path,
-        ivy = detail._2._2._2._2._1.path,
-        artifact = detail._2._2._2._2._2,
-        extras = Seq()
-      )
```

Comment:
The idea of publishing also doc and sources is to ensure we simulate the real use case as much as possible. Also, it ensures, that docJar and sources jar could be sucessfully created when we claim that tests have passed.

Reply:
If user want the ensuring "..claim that tests have passed" then he/she will use a `TestInvocation` to ensure that. Eg `Seq(Targets(Seq("foo.publishLocal")), Targets(Seq("verify")))`
Note that, by default, mill publishLocal to `os.home / ".ivy2" / "local"` so the .jar files we create are not path-conflict with that.